core/feeders/general.json

```
@@ -3,92 +3,110 @@
 		{
 			"name": "HackerNoon",
 			"link": "https://hackernoon.com/feed",
-			"web": "https://www.hackernoon.com/"
+			"web": "https://www.hackernoon.com/",
+			"category": "Programming"
 		},
 		{
 			"name": "Producthunt",
 			"link": "https://www.producthunt.com/feed",
-			"web": "https://www.producthunt.com/"
+			"web": "https://www.producthunt.com/",
+			"category": "Software Development"
 		},
 		{
 			"name": "freeCodeCamp",
 			"link": "https://www.freecodecamp.org/news/rss/",
-			"web": "https://www.freecodecamp.org/news/"
+			"web": "https://www.freecodecamp.org/news/",
+			"category": "Programming"
 		},
 		{
 			"name": "opensource",
 			"link": "https://opensource.com/feed",
-			"web": "https://opensource.com/"
+			"web": "https://opensource.com/",
+			"category": "Programming"
 		},
 		{
 			"name": "changelog",
 			"link": "https://changelog.com/feed",
-			"web": "https://changelog.com/"
+			"web": "https://changelog.com/",
+			"category": "Software Development"
 		},
 		{
 			"name": "itsfoss",
 			"link": "https://itsfoss.com/category/news/feed",
-			"web": "https://itsfoss.com/"
+			"web": "https://itsfoss.com/",
+			"category": "Programming"
 		},
 		{
 			"name": "thenewstack",
 			"link": "https://thenewstack.io/feed",
-			"web": "https://thenewstack.io/"
+			"web": "https://thenewstack.io/",
+			"category": "Programming"
```

```suggestion
			"category": "Technology"
```

The NewStack is a general tech website, just like "The Verge"